### PR TITLE
Complete i18n

### PR DESCRIPTION
**Pull request by stephdewit**
Hello.

I translated a few components which weren't yet.

I only checked the main view and the torrent details modal, not the `Settings` and `RSS` views. But it's a beginning. 😄

Sorry for the big PR, maybe I should have done multiple smaller ones. I got carried away! 🐵

### CODE REVIEW

**Comment by jfurrow** [comment]
Thanks for this! Unfortunately I merged another PR which caused merge conflicts here, sorry about that.

Also, sorry for my absence, I've been traveling but am home now! I'll review this tomorrow.

**Comment by stephdewit** [comment]
I hope you enjoyed your travel!

I fixed the conflicts.

File: client/scripts/util/formatData.js [hunk 1]
```diff
@@ -1,127 +1,6 @@
...
-    if (extraUnits) {
-      unit += extraUnits;
-    }
-
-  	return {
-  		value,
-  		unit
-  	};
-  },
-
```
Comment (jfurrow) [inline client/scripts/util/formatData.js]: 💯 I'm glad to see this go!

**Comment by jfurrow** [comment]
This is really, really nice. Thanks so much for doing this!

**Pull request merged**